### PR TITLE
Refactor SQR fields

### PR DESCRIPTION
**Pull request by DadImScared**
https://trello.com/c/mfrh1aWH/235-refactor-sqr-fields

### CODE REVIEW

File: frontend/src/components/SQRForm/Fields.vue [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+          pathOverride="comments"
+          :fieldProps="{
+            filled: true,
+            outlined: true,
+            required: true
+          }"
+        ></text-area>
+        <v-expansion-panels class="pt-3">
+          <v-expansion-panel>
+            <v-expansion-panel-header class="pl-0">Guidelines</v-expansion-panel-header>
```
Comment (anantakrishna) [inline frontend/src/components/SQRForm/Fields.vue]: I have an idea. Why not to extract `Guidelines` component with a slot for content?

File: frontend/src/views/SQR/Form.vue [hunk 2]
```diff
@@ -143,117 +94,12 @@ enum SubmissionsBranch {
 
 @Component({
   name: "Form",
-  components: { SQRField, UnwantedParts, SoundIssues, Duration, TextArea },
+  components: { Fields, CancelListItem },
```
Comment (anantakrishna) [inline frontend/src/views/SQR/Form.vue]: What is the rationale behind extracting Fields to a separate component? I guess it is used only once, here.

Comment (DadImScared) [reply frontend/src/views/SQR/Form.vue]: It's easier to read as a separate component.

Comment (anantakrishna) [reply frontend/src/views/SQR/Form.vue]: OK then.

**Pull request merged**